tools/topology/topology2/include/components/asrc.conf

```diff
@@ -94,6 +119,8 @@ Class.Widget."asrc" {
 	# Default attributes for asrc
 	#
 	type		"asrc"
-	uuid 		"f6:72:ec:c8:26:85:af:4f:9d:39:a2:3d:0b:54:1d:e2"
+	uuid 		"2d:40:b4:66:68:b4:f2:42:81:a7:b3:71:21:86:3d:d4"
```

Comment:
This is a bit odd, why does the UUID change?

Reply:
Done, thanks!